targets/liteos-stm32f4/README.md

```diff
@@ -0,0 +1,67 @@
+### About
+
+This folder contains files to run JerryScript on LiteOS with STM32F4-Discovery board.
+
+### How to build
+
+#### 1. Preface
+
+1, Directory structure
+
+Assume `jerry-liteos` as the path to the projects to build.
+The folder tree related would look like this.
+
+```
+jerry-liteos
+  + jerryscript
+  |  + targets
+  |      + liteos-stm32f4
+  + LiteOS_Kernel
+```
+
+2, Target board
+
+Assume [STM32F4-Discovery with BB](http://www.st.com/web/en/catalog/tools/FM116/SC959/SS1532/LN1199/PF255417)
+as the target board.
+
+#### 2. Prepare LiteOS
+
+Follow [this](https://github.com/LITEOS/LiteOS_Kernel) page to get the LiteOS source.
```

Comment:
Gcc version should be 5.4 and above. If the version is too low. There is a #error message.